src/templates/fr/2020/methodology.html

```diff
@@ -360,39 +365,85 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
+          </li>
+          <li>
+            Les responsables du projet ont examiné toutes les nominations d’auteurs et d’autrices et se sont efforcés de sélectionner des personnes qui apportent de nouvelles perspectives et amplifient les voix des groupes sous-représentés dans la communauté.
+          </li>
+        </ul>
+
+        Nous espérons réitérer ce processus à l’avenir afin de garantir que le Web Almanac soit un projet plus diversifié et inclusif avec des contributeurs et contributrices de tous horizons.
       </p>
 
       <p>
-        Nous avons passé le mois de juillet 2020 à apparier les personnes avec les chapitres et à recueillir leurs commentaires pour finaliser les indicateurs particuliers qui constitueront chaque chapitre.
+        Enfin, en juillet 2020, après des cycles de brainstorming et de nominations, 22 chapitres ont été consolidés et nous avons formé des équipes chargées de la rédaction, de la révision et de l’analyse du contenu de chaque chapitre.
       </p>
     </section>
 
     <section>
       <h3 id="analysis"><a href="#analysis" class="anchor-link">Analyse</a></h3>
 
       <p>
-        En juillet et août 2020, ayant une liste bien établie de mesures et de chapitres, les analystes de données ont trié les mesures pour en déterminer la faisabilité. Dans certains cas, il a fallu créer des <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">mesures personnalisées</a> pour combler les lacunes de nos capacités analytiques.
+        En juillet et août 2020, ayant une liste bien établie de mesures et de chapitres, les analystes de données ont trié les mesures pour en déterminer la faisabilité. Dans certains cas, nous avons créé des <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">mesures personnalisées</a> pour combler les lacunes de nos capacités analytiques.
       </p>
 
       <p>
         Tout au long du mois d’août 2020, le pipeline de données HTTP Archive a parcouru plusieurs millions de sites web, rassemblant les métadonnées à utiliser dans le Web Almanac.
       </p>
 
       <p>
-        Les analystes de données ont commencé à rédiger des requêtes pour extraire les résultats de chaque mesure. Au total, les <strong> requêtes TBC</strong> ont été écrites à la main ! Vous pouvez parcourir toutes les requêtes par chapitre dans le répertoire <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> du dépôt GitHub du projet.
+        Les analystes de données ont commencé à rédiger des requêtes pour extraire les résultats de chaque mesure. Au total, <strong>TBC queries</strong> ont été écrites à la main&nbsp;! Vous pouvez parcourir toutes les requêtes par chapitre et année dans le <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">dépôt des requêtes</a> sur GitHub.
```

Comment:
lang="en" sur TBC queries

Reply:
I'm not going to use it because, if I understand correctly, it is a temporary placeholder. It will be replaced at the end by the real number of requests and this number will not be an english word

Reply:
@borisschapira can you check this again? https://github.com/HTTPArchive/almanac.httparchive.org/pull/1449/ changed it to the following, so think you missed this update:

![Changes in 1449 to this sentence](https://user-images.githubusercontent.com/10931297/98919921-5fd7c480-24c7-11eb-8ecb-57d33fcb8489.png)

Reply:
Oh damn right. Fixed.